File: aoBlogs2/Models/BlogViewingLogModel.vb


Option Explicit On
Option Strict On

Imports System
Imports System.Collections.Generic
Imports System.Text
Imports Contensive.BaseClasses

Namespace Models
    Public Class BlogViewingLogModel        '<------ set set model Name and everywhere that matches this string
        Inherits DbModel
        '
        '====================================================================================================
        '-- const
        Public Const contentName As String = "Blog Viewing Log"      '<------ set content name
        Public Const contentTableName As String = "BlogViewingLog"   '<------ set to tablename for the primary content (used for cache names)
        Private Shadows Const contentDataSource As String = "default"             '<------ set to datasource if not default
        '
        '====================================================================================================
        ' -- instance properties
        'instancePropertiesGoHere
        Public Property BlogEntryID As Integer
        Public Property MemberID As Integer
        Public Property VisitID As Integer
        '

    End Class
End Namespace

File: source/aoBlogs2/Models/Db/BlogViewingLogModel.cs


using Contensive.Models.Db;

namespace Contensive.Blog.Models {
    public class BlogViewingLogModel : Contensive.Models.Db.DbBaseModel {        // <------ set set model Name and everywhere that matches this string
        // 
        // ====================================================================================================
        public static DbBaseTableMetadataModel tableMetadata { get; private set; } = new DbBaseTableMetadataModel("Blog Viewing Log", "BlogViewingLog", "default", false);
        // -- const
        // 
        // ====================================================================================================
        // -- instance properties
        // instancePropertiesGoHere
        public int BlogEntryID { get; set; }
        public int MemberID { get; set; }
        public int VisitID { get; set; }
        // 

    }
}